test/test_operations.py

```diff
@@ -2172,6 +2624,10 @@ def test_send_to_device_grad(self):
     dt = xm.send_cpu_data_to_device([t], xla_device)
     self.assertTrue(dt[0].requires_grad)
 
+  @unittest.skipIf(
+      os.environ.get('PYTORCH_XLA_TESTS_SKIP'),
+      'To avoid new feature developing, disable failed PyTorch/XLA test on TPUVM'
```

Comment:
hmm, Are all these tests failing on TPUVM? that sounds like a huge issue.

Reply:
yes, many opbyop tests failed: https://pantheon.corp.google.com/cloud-build/builds/f3d7c450-5147-405e-b1a8-e53f86e96ecb?project=tpu-pytorch&jsmode=o&mods=allow_workbench_image_override

with almost the same error message for these failed tests:
```
Traceback (most recent call last):
     File "/pytorch/xla/test/test_operations.py", line 703, in test_random_from_to_bool
     self.assertTrue(from_val <= x.to(torch.int).min() < (from_val + delta))
 RuntimeError: ./tensorflow/compiler/xla/xla_client/pjrt_computation_client.h:106 : ExecuteChained not implemented
```

Reply:
pjrt does not support op by op, we can skip `run_opbyop` for PJRT in https://github.com/pytorch/xla/blob/master/test/run_tests.sh#L138.

Maybe in `run_opbyop` add a check for `PJRT_DEVICE`

Reply:
sg, will do

Reply:
will disable `run_async_scalar` too, because many tests failed on TPUCI too with the almost same reason:
https://pantheon.corp.google.com/cloud-build/builds/617849b5-377c-4536-bcb4-ea1a4abf99f0?project=tpu-pytorch&jsmode=o&mods=allow_workbench_image_override

Reply:
will use `PJRT_DEVICE` to check after this trigger's result

Reply:
done

Reply:
If these are only failing in `test_opbyop`, then can you remove these skips? We run test_operations multiple times with different settings, and we don't want to skip them when we have settings where we expect them to work